responses/en/HassMediaPrevious.yaml

```diff
@@ -0,0 +1,5 @@
+language: en
+responses:
+  intents:
+    HassMediaPrevious:
+      default: "Going back"
```

Comment:
I think this sounds ambiguous. Maybe "Playing previous", for consistency with `HassMediaNext`?